src/routers/routerV2.ts

```diff
@@ -1,12 +1,24 @@
 import { Router } from 'express';
+import rateLimit from 'express-rate-limit';
 import { getWords, getWord } from '../controllers/words';
 import { getExample, getExamples } from '../controllers/examples';
 import { getNsibidiCharacter, getNsibidiCharacters } from '../controllers/nsibidi';
 import { getTranscription } from '../controllers/speechToText';
+import { getTranslation } from '../controllers/translation';
 import validId from '../middleware/validId';
 import validateApiKey from '../middleware/validateApiKey';
 import analytics from '../middleware/analytics';
 import attachRedisClient from '../middleware/attachRedisClient';
+import { MiddleWare } from 'src/types';
+
+// TODO: add rate limiting with upstash
+// const ONE_DAY = 24 * 60 * 60 * 1000;
+// const REQUESTS_PER_MS_TRANSLATION = 5;
+
+// const translationRateLimiter: MiddleWare = rateLimit({
+//   windowMs: ONE_DAY,
+//   max: REQUESTS_PER_MS_TRANSLATION,
+// });
```

Comment:
you can delete this comment since you have a corresponding GitHub issue

Reply:
done!